projects/1.18/assets/italian-delights-add-on-for-farmers-delight/italian_delight/lang/zh_cn.json

```
@@ -0,0 +1,50 @@
+{
+  "advancements.covermeinwine.descr": "制作你的第一份红酒萨拉米！",
+  "advancements.covermeinwine.title": "红酒裹身！",
+  "advancements.dragon_borne.descr": "喝下龙裔酒。",
+  "advancements.dragon_borne.title": "你是……龙裔！",
+  "advancements.first_pizza.descr": "制作你的第一份披萨！",
+  "advancements.first_pizza.title": "披萨时间！",
+  "block.italian_delight.crop_0": "葡萄作物",
+  "block.italian_delight.crop_1": "葡萄作物",
+  "block.italian_delight.crop_2": "葡萄作物",
+  "block.italian_delight.crop_3": "葡萄作物",
+  "block.italian_delight.crop_4": "葡萄作物",
+  "block.italian_delight.crop_5": "葡萄作物",
+  "block.italian_delight.crop_6": "葡萄作物",
+  "block.italian_delight.crop_7": "成熟的葡萄作物",
+  "block.italian_delight.ham_pizza_placed": "放置的火腿披萨",
+  "block.italian_delight.muchroom_pizza_block": "蘑菇披萨快",
+  "block.italian_delight.pizza_diavola_placed": "放置的魔鬼披萨",
+  "block.italian_delight.pizza_margherita": "玛格丽塔披萨",
+  "block.italian_delight.pizza_margherita_1quarter": "四分之一份玛格丽塔披萨",
+  "block.italian_delight.woode_oven": "木质烤炉",
+  "item.italian_delight.dragonborn_wine": "龙裔酒",
+  "item.italian_delight.glowberry_wine": "发光浆果酒",
+  "item.italian_delight.grape_bunch": "葡萄串",
+  "item.italian_delight.grape_seeds": "葡萄种子",
+  "item.italian_delight.ham_pizza": "熟火腿披萨",
```

```suggestion
  "item.italian_delight.ham_pizza": "火腿披萨",
```
统一性，也可以下面的这类物品和放出来的方块前面都加个`熟`
不过菜名一般不这样吧（）